TrafficCapture/dockerSolution/src/main/docker/migrationConsole/createProcessSnapshotWorkflow.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Note, really don't love `yaml` files getting scattered throughout the codebase, is there another extension we can use to differentiate argo vs k8 vs helm vs other declarative files?